### ISSUE TITLE
v2.formatReserves doesn't contain the incentivesAPY data

### ISSUE THREAD

**Issue by GalloDaSballo** [aave/aave-js#169]
If you check the source code for `v2.formatReserves` you can see it will return
```javascript
      aIncentivesAPY,
      vIncentivesAPY,
      sIncentivesAPY,
```

However, these keys are not returned in the npm distributed package

# Demo

## NextJS Code
```typescript
  // Graphql Request for Reserve Data
  const reserve = useReserve(
    "0x0d500b1d8e8ef31e21c99d1db9a6444d3adf12700xd05e3e715d945b59290df0ae8ef85c1bdb684744"
  );
  console.log("reserve", reserve);

  // Apr for depoist and borrow is in aaveRes

  const aaveRes: any = reserve
    ? v2.formatReserves([reserve as any], new Date().getTime() / 1000)
    : null;

```

## Result
```javascript
aToken: {__typename: "AToken", id: "0x8df3aad3a84da6b69a4da8aec3ea40d9091b2ac4"}
availableLiquidity: "152281094.315379813719490558"
averageStableRate: "0"
avg30DaysLiquidityRate: undefined
avg30DaysVariableBorrowRate: undefined
baseLTVasCollateral: "0.5"
baseVariableBorrowRate: "0"
borrowingEnabled: true
decimals: 18
id: "0x0d500b1d8e8ef31e21c99d1db9a6444d3adf12700xd05e3e715d945b59290df0ae8ef85c1bdb684744"
isActive: true
isFrozen: false
lastUpdateTimestamp: 1620052295
liquidityIndex: "1.00278985632621353962"
liquidityRate: "0.04572907692833141764"
name: "Wrapped Matic"
optimalUtilisationRate: "450000000000000000000000000"
price: {__typename: "PriceOracleAsset", priceInEth: "0.00025805"}
reserveFactor: "0.2"
reserveLiquidationBonus: "0.1"
reserveLiquidationThreshold: "0.65"
sToken: {__typename: "SToken", id: "0xb9a6e29fb540c5f1243ef643eb39b0acbc2e68e3"}
stableBorrowRate: "0.20427708837168427978"
stableBorrowRateEnabled: false
stableDebtLastUpdateTimestamp: 0
stableRateSlope1: "100000000000000000000000000"
stableRateSlope2: "3000000000000000000000000000"
symbol: "WMATIC"
totalDebt: "129589585.171412279915014675"
totalLiquidity: "281870679.486792093634505233"
totalPrincipalStableDebt: "0"
totalScaledVariableDebt: "128617818.93323034014392991"
totalStableDebt: "0"
totalVariableDebt: "129589585.171412279915014675"
underlyingAsset: "0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270"
usageAsCollateralEnabled: true
utilizationRate: "0.45974836902993520944"
vToken: {__typename: "VToken", id: "0x59e8e9100cbfcbcbadf86b9279fa61526bbb8765"}
variableBorrowIndex: "1.0075554121260884871"
variableBorrowRate: "0.12427708837168427978"
variableRateSlope1: "70000000000000000000000000"
variableRateSlope2: "3000000000000000000000000000"
```

## Recommended action
I believe you just need to compile locally and re-publish

**Comment by sakulstra** [comment]
Hello @GalloDaSballo, we didn't release this yet on purpose - the issue is that this introduced a breaking change which was not our intention. Therefore we're working on a new version reverting that breakage https://github.com/aave/aave-js/pull/153/files, but didn't yet get to finish it yet.

---

If you need this asap, you can always also use the canary/branch releases (check the action log of a certain pr) https://github.com/aave/aave-js/pull/152 -> https://github.com/aave/aave-js/runs/2390396454?check_suite_focus=true#step:8:46  -> `2.7.3-799111eeb828c0d33b8dacc6ecb04fc8736a3bff.0`

**Comment by GalloDaSballo** [comment]
Hey @sakulstra! Thank you for the advice!
It seem like the current code returns "NaN"

I'm sure you're already working on this

<img width="220" alt="Screenshot 2021-05-03 at 17 02 54" src="https://user-images.githubusercontent.com/13383782/116893573-6868b680-ac31-11eb-98fc-549d05e60339.png">


Let me know if I can help speed up this release!

**Comment by sakulstra** [comment]
It shouldn't return null :) aave.com uses it and seems to work.

To correctly calculate the incentive apy in it's current state you have to make sure to query all the related fields from graphql:
https://github.com/aave/aave-js/pull/128/files#diff-07006b4d58b3fbb84f6ceab251c157f1adbfd20be6366ced86a8eb284d2c231eR48-R56

And pass in the correct reward token + rewardTokenPrice:
https://github.com/aave/aave-js/pull/128/files#diff-07006b4d58b3fbb84f6ceab251c157f1adbfd20be6366ced86a8eb284d2c231eR93-R98

As you can't fetch stkAave from thesubgraph i'd suggest to use aave instead - same for WMATIC, just use MATIC instead.
Will let you know & update comments/docs once this is finalized.

**Comment by GalloDaSballo** [comment]
Thank you @sakulstra!

**Comment by sakulstra** [comment]
@GalloDaSballo, sorry we didn't yet get to making this unbreaking so instead of postponing further we released a 3.0 to signal the breaking change. Make sure to update in case you're on the beta as we pushed some fixes to incentive calculation.
